p2panda-rs/src/storage_provider/traits/requests.rs

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use crate::document::DocumentId;
+use crate::entry::EntrySigned;
+use crate::identity::Author;
+use crate::operation::OperationEncoded;
+use crate::storage_provider::ValidationError;
+use crate::Validate;
+
+/// Request body of `panda_getEntryArguments`.
+pub trait AsEntryArgsRequest {
```

Comment:
Could name this and the other request/response traits with an imperative like `Validate` or `Copy`: `RequestEntryArgs` / `RespondEntryArgs`

Reply:
Hmmm... yeh that is nice.

Reply:
But its not really an action, isn't it? There is nothing "returning" in that trait (like "validating" or "copying") but only definining the shape of what is returned. I'd keep the trait name as is :+1:

Reply:
I'm also fine with the current name. I felt it would be nice to use a name that doesn't just wrap the name of our standard implementation.